pyfixest/did/lpdid.py

```diff
@@ -163,7 +166,8 @@ def iplot(
         df = self._coeftable
         df["fml"] = "lpdid"
 
-        return _coefplot(
+        _coefplot(
```

Comment:
The return statement is missing. This leads to the error when building the docs in the CI: 

```python
import pandas as pd
from pyfixest.did.estimation import lpdid

url = "https://raw.githubusercontent.com/s3alfisc/pyfixest/master/pyfixest/did/data/df_het.csv"
df_het = pd.read_csv(url)

fit = lpdid(
    df_het,
    yname="dep_var",
    idname="unit",
    tname="year",
    gname="g",
    vcov={"CRV1": "state"},
    pre_window=-20,
    post_window=20,
    att=False
)

fit.tidy().head()
fit.iplot(figsize= [1200, 400], coord_flip=False).show()
------------------


---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[1], line 20
      7 fit = lpdid(
      8     df_het,
      9     yname="dep_var",
   (...)
     16     att=False
     17 )
     19 fit.tidy().head()
---> 20 fit.iplot(figsize= [1200, 400], coord_flip=False).show()

AttributeError: 'NoneType' object has no attribute 'show'
```

Reply:
Fixed 👍